Ginger/GingerCore/Environments/Database.cs

```diff
@@ -394,7 +394,10 @@ public Boolean Connect(bool displayErrorPopup = false)
 
                     case eDBTypes.Cassandra:
                         GingerCassandra CassandraDriver = new GingerCassandra(this);
-                        CassandraDriver.Connect();
+                        bool cassandraConnection;
+                        cassandraConnection= CassandraDriver.Connect();
+                        if (cassandraConnection == true)
```

Comment:
If connected successfully then you need to update last connection used time
LastConnectionUsedTime = DateTime.Now;